lib/caterpillar-ruby/version.rb

```diff
@@ -1,3 +1,3 @@
 module Caterpillar
-  VERSION = '0.2.0'
+  VERSION = '0.3.0'
```

Comment:
Since this is a bug fix, I would recommend increasing only the patch number and make this `0.2.1`.